dev.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:20-slim
+FROM --platform=arm64  node:20-slim
```

Comment:
Q: what does this do in terms of making this "cross platform"?

Reply:
From what I know, by setting --platform=arm64, it supports all devices with 64-bit processing. By default, Docker figures out and uses the same platform as the host. So here, we are hardcoding it. Since most devices now have 64-bit processing, I am adding this just to ensure it works on all devices.